File: frontend/scripts/parse_screens.py
import json

with open('stitch_screens.json', 'r') as f:
    data = json.load(f)

text_content = data['result']['content'][0]['text']
screens_data = json.loads(text_content)

for screen in screens_data['screens']:
    print(f"Name/ID: {screen.get('name')}")
    print(f"Title: {screen.get('title')}")
    html_code = screen.get('htmlCode', {})
    if 'downloadUrl' in html_code:
        print(f"Has HTML URL: True")
    print("---")

File: frontend/scripts/parse_screens.js
const fs = require('fs');

const data = JSON.parse(fs.readFileSync('stitch_screens.json', 'utf8'));
const textContent = data.result.content[0].text;
const screensData = JSON.parse(textContent);

console.log("Total screens:", screensData.screens.length);
screensData.screens.forEach(screen => {
  console.log("Name:", screen.name);
  console.log("Title:", screen.title);
  if (screen.htmlCode && screen.htmlCode.downloadUrl) {
    console.log("Has HTML: Yes");
  } else {
    console.log("Has HTML: No");
  }
  console.log("---");
});
